core/runtime/src/main/java/io/quarkus/runtime/Quarkus.java

```diff
@@ -1,20 +0,0 @@
-package io.quarkus.runtime;
-
-import java.lang.reflect.Method;
-
-/**
- * The main entry point class, calling main allows you to bootstrap quarkus
- * <p>
- * Note that at native image generation time this is replaced by {@link io.quarkus.runtime.graal.QuarkusReplacement}
- * which will avoid the need for reflection.
- * <p>
- * TODO: how do we deal with static init
- */
-public class Quarkus {
```

Comment:
These classes are unused?

Reply:
Yes, looks so to me. I'd like another opinion though @stuartwdouglas @dmlloyd

Reply:
I believe that the intent of this class was to provide a class that can be run from an IDE.

Reply:
Oh good to know, thanks David. So we should keep it. Same question for `QuarkusReplacement.java` contains a substitution of the generated main class? Can it be deleted?

Reply:
I'm not sure what the current status of that is TBH.

Reply:
I’ll try removing it and keep the Substitutee and see if it the substitutions are still needed.

Reply:
I kept the `Quarkus` for IDE runs and removed the `QuarkusReplacement`.

Reply:
These are leftovers from the early days, they can both go.

Reply:
So the two classes are gone. This is ready for another round of review.